parquet-column/src/main/java/parquet/vector/VectorizedColumnReaderImpl.java

```diff
@@ -490,7 +489,15 @@ public void readVector(ColumnVector vector) {
       if (!valueRead) {
         binding.readVector(vector);
         valueRead = true;
-        this.pages = null; //processed this batch
+//        this.pages = null; //processed this batch
+        long totalValueCount = 0;
```

Comment:
there is already a:
private final long totalValueCount;
in the class, rename it to valueCountInPages or something?